package.json

```diff
@@ -15,7 +15,7 @@
     "type-check:cypress": "tsc --project cypress/tsconfig.json",
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx --max-warnings 0 --report-unused-disable-directives",
     "lint:fix": "npm run lint -- --fix",
-    "prettier-base": "prettier . --ignore-path .gitignore --ignore-unknown",
+    "prettier-base": "prettier . --ignore-path .gitignore --ignore-unknown --cache",
```

Comment:
You can do that with [the `--logLevel` flag](https://prettier.io/docs/en/cli.html#--loglevel):

`npm run format -- --logLevel silent`

Reply:
Can we add the `--logLevel silent` flag to the command, or will that cause problems?

Reply:
I'd avoid that. Otherwise, we won't see which files have an invalid format in CI. Instead, we can set the flag to `--loglevel warn` so files processed are not logged but files with warnings or errors appear in the output.

Reply:
```suggestion
    "prettier-base": "prettier . --ignore-path .gitignore --ignore-unknown --cache --loglevel warn",
```

Like this?

Reply:
Yes! But without the `ignore-path` flag I removed recently.